gridy/Sources/Presentations/PlanBoard/ShareImageView.swift

```diff
@@ -2,32 +2,259 @@
+    @State private var isEndDateHovered = false
+    @State private var selectedStartDate = Date()
+    @State private var isStartDatePickerPresented = false
+    @State private var selectedEndDate = Date()
+    @State private var isEndDatePickerPresented = false
+    
+    @Environment(\.colorScheme) var colorScheme
+    @State var capture: NSImage?
     
     var body: some View {
-        NavigationStack {
-            Text("ShareImageView")
-                .scenePadding()
-        }
-        .toolbar {
-            ToolbarItem(placement: .cancellationAction) {
-                Button("Cancel") {
+        VStack(spacing: 16) {
+            RoundedRectangle(cornerRadius: 8)
+                .overlay {
+                    if ImageRenderer(content: TimelineLayoutView()).nsImage != nil {
+                        Image(nsImage: ImageRenderer(content: TimelineLayoutView()).nsImage!)
+                            .resizable()
+                            .scaledToFit()
+                    } else {
+                        Text("Thumbnail")
+                            .foregroundStyle(Color.subtitle)
+                    }
+                }
+                .foregroundStyle(Color.item)
+                .frame(width: 436, height: 330)
+            RoundedRectangle(cornerRadius: 8)
+                .overlay {
+                    VStack(alignment: .leading) {
+                        HStack(spacing: 8) {
+                            RadioButton(isSelected: periodSelection == .allPeriods)
+                                .onTapGesture { periodSelection = .allPeriods }
+                                .padding(.leading, 16)
+                            Text("기간 모두")
+                                .foregroundStyle(Color.title)
+                        }
+                        HStack {
+                            RadioButton(isSelected: periodSelection == .setPeriods)
+                                .onTapGesture { periodSelection = .setPeriods }
+                                .padding(.leading, 16)
+                            Text("기간 설정")
+                                .foregroundStyle(Color.title)
+                            Spacer()
+                                .frame(width: 16)
+                            Button(action: {
+                                isStartDatePickerPresented = true
+                            }) {
+                                Text("시작 : \(formattedDate(date: selectedStartDate))")
+                                    .foregroundStyle(Color.title)
+                                    .frame(width: 132, height: 32)
+                                    .background(isStartDateHovered ? Color.itemHovered : Color.clear)
+                                    .cornerRadius(5)
+                                    .onHover { hover in
+                                        isStartDateHovered = hover
+                                    }
+                            }
```

Comment:
```suggestion
                            Button {
                                isStartDatePickerPresented = true
                            } label: {
                                Text("시작 : \(formattedDate(date: selectedStartDate))")
                                    .foregroundStyle(Color.title)
                                    .frame(width: 132, height: 32)
                                    .background(isStartDateHovered ? Color.itemHovered : Color.clear)
                                    .cornerRadius(5)
                                    .onHover { hover in
                                        isStartDateHovered = hover
                                    }
                            }
```
이렇게 스타일을 맞추고 있습니다 ~. ~

Reply:
다른 버튼도 전부 수정 부탁해요!